lib/kafo/hook_context.rb

```diff
@@ -134,5 +134,10 @@ def scenario_path
     def scenario_data
       self.kafo.config.app
     end
+
+    # Return the current exit code
+    def exit_code
+      self.kafo.exit_code
+    end
```

Comment:
For my benefit, could you please confirm my understand of what this does?

1. Kafo initializes with an exit code of 0. At any point if an issue is encountered, that value can be changed

2. This method allows us to check the current value of the exit code within a hook

Reply:
That is correct. Today this is used by a post hook and checks the value directly e.g. `@kafo.exit_code` rather than using a method that is within the context itself, i.e. `exit_code`